src/main/kotlin/com/chillibits/coronaaid/repository/DistrictRepository.kt

```diff
@@ -0,0 +1,18 @@
+package com.chillibits.coronaaid.repository
+
+import com.chillibits.coronaaid.model.db.District
+import com.chillibits.coronaaid.model.dto.DistrictAnalyticsDto
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
+
+interface DistrictRepository: JpaRepository<District, Int> {
+
+    @Query(
```

Comment:
This SQL query is pretty performance intensive. I will improve this with next performance update.